_posts/2018-08-15-JS-SDK.md

```diff
@@ -57,15 +57,14 @@ With the modern SDK you get the equivalent with just a fraction of the lines:
 ```js
 async function loginAndGetFeed(username, password,) {
   await Parse.User.logIn(username, password);
-  const feed = await new Parse.Query('Feed').find();
-  return feed;
+  return new Parse.Query('Feed').find();
```

Comment:
wanted to make it clear that you get the feed out of it, and then return it. 

Here it looks like you're returning a 'simple' promise.

Reply:
Well, in either case, its the same.  Also, we're promoting fewer lines as a benefit, this is fewer.

Having said that, I trust your judgment, so go with what you think is best.

Reply:
I know it's exactly the same. I don't have the energy to change it back so let's go with this solution.

Just wanted to point out that I wrote it that way on purpose, to show that you get a list of ParseObject as a result when one `await Query.find()`.

Many users are still using the backbone API, most havent heard of async / await. :)